Week0/README.md

```diff
@@ -2,36 +2,11 @@
 
 ```
 In week one we will discuss the following topics:
-• The first part of the day Unmesh will talk about CLI (Command Line Interface)
-• Intro JavaScript (What is it, where can you use it for)
-• Variables [var, let, const]
-• Basic Data types [Strings, Numbers, Arrays, Booleans]
-• Operators
+• Git
 ```
 
+### Here are resources that we like you to read as a preparation for the coming lecture. 
```

Comment:
Now that Node version 8 LTS is out, I would recommend that, for the time being, we ask students to install the LTS version. This already has all the ES6 features we want. Last time I checked there were still NPM packages that complained about version 9.